module-1/cdk/lib/Module1Construct.ts

```diff
@@ -0,0 +1,74 @@
+import cloudfront = require("@aws-cdk/aws-cloudfront");
+import iam = require("@aws-cdk/aws-iam");
+import s3 = require("@aws-cdk/aws-s3");
+import s3deploy = require("@aws-cdk/aws-s3-deployment");
+import cdk = require("@aws-cdk/cdk");
+
+export class Module1Construct extends cdk.Construct {
+  constructor(scope: cdk.Construct, id: string, contentDirectory: string) {
```

Comment:
The recommended pattern is to always use a `props` argument as the third argument. This is effectively how you model keyword arguments for functions in JavaScript/TypeScript

Reply:
Will refactor. https://github.com/DavidChristiansen/aws-modern-application-workshop/issues/31